### ISSUE TITLE
Adding a "filelist" content element throws an exception

### ISSUE THREAD

**Issue by erredeco** [FriendsOfTYPO3/frontend_editing#400]
Frontend_editing: 1.9.0
TYPO3: 9.5.13

If I try to add a "File list" content element (on both backend or frontend) I get an exception:

```
(1/2) TYPO3Fluid\Fluid\Core\Parser\UnknownNamespaceException
Fluid parse error in template Standard_action_Uploads_97f0e33bd377f3c89bbfaf7aa48fdc045fd6512a, line 11 at character 1. Error: Unknown Namespace: t3kit (error code 0). Template source chunk: <t3kit:render.svg aria-label="{f:translate(key:'LLL:EXT:lang/Resources/Private/Language/locallang_tca.xlf:sys_file.type')}: {file.extension}" source="{f:uri.resource(path: 'Icons/FileIcons/{file.extension}.svg', extensionName: 'theme_t3kit')}"/>
```

**Comment by mabolek** [comment]
Hi @erredeco! 

Thank you for submitting. 

This looks like it is related to a t3kit template, I'm unable to reproduce this in my t3kit test install.

Does the error go away if you uninstall Frontend Editing?

**Comment by erredeco** [comment]
It is not reproducible without EXT:frontend_editing; I don't understand where "t3kit" comes from as I don't use any "t3kit" template; the only occurrences I have found are:

```
public/typo3conf/ext/frontend_editing/Resources/Private/Templates/FluidStyledContent9/Uploads.html
public/typo3conf/ext/frontend_editing/ext_localconf.php
```
So I have seen that it comes out when I added "Editable fluid styled content V9" on TS template > Include static from extensions. Using just "Frontend Editing", it works.

What is the difference between "Frontend Editing" and "Editable fluid styled content V9" ?

**Comment by mabolek** [comment]
@erredeco I meant Fluid template, not t3kit template. Sorry. I was unable to reproduce it on my t3kit install, which should be using the same template.

"Frontend Editing" is the TypoScript template that enables Frontend Editing. It should always be loaded.

"Editable fluid styled content V9" loads the editable TypoScript templates for Fluid Styled Content. This one is optional. It may be affected by custom templates. If you don't have any custom Fluid templates, let me know! Also, check out https://docs.typo3.org/p/friendsoftypo3/frontend-editing/master/en-us/Installation/Index.html#optional-steps

**Comment by erredeco** [comment]
Yes I have some custom templates, I will try to look at them, but they all worked with jsut "Frontend Editing" TypoScript. The error says "extensionName: 'theme_t3kit'" which I don't own

**Comment by mabolek** [comment]
@erredeco Test if it works with the branch `401-adding-afilelist-ce-throws-exception`. I think the Uploads.html template shouldn't be there at all. It has no editable areas.

PR #404

**Comment by erredeco** [comment]
> @erredeco Test if it works with the branch `401-adding-afilelist-ce-throws-exception`. I think the Uploads.html template shouldn't be there at all. It has no editable areas.
> 
> PR #404

It seemed to help; question: the "inline" editing of the text editor should work only with the "Text" element? It does not seem to work with "Textpic" or "Textmedia"

**Comment by mabolek** [comment]
@erredeco Thanks for checking the pull request. I'll merge it.

As for the editability of the Text element. It's _supposed_ to work, so feel free to open an issue about the problem.

**Comment by mabolek** [comment]
Fixed in #404
